src/MigrationVersions/content_type_group.yaml

```diff
@@ -8,9 +8,13 @@
 -
     type: content_type_group
     mode: create
-    content_type_group: 1
     identifier: Professionals
 
+-
+    type: content_type_group
```

Comment:
What is this new group for?

Reply:
This group is now useless and i removed it.